### ISSUE TITLE
[Bug] [1.19.2] Feature order cycle crash

### ISSUE THREAD

**Issue by Varlanou** [quiqueck/BetterEnd#186]
### What happened?

Tried this a number of times, with new worlds and everything. It seems that trying to enter the End is throwing back some errors due to a feature order cycle. This and BYG are the only mods I have messing with End biome generation. Below is the error I received.

Tested this with both 0.75.1 and 0.76.0 Fabric API.

Also tested with and without a resource pack active.

### BetterEnd

2.1.4

### BCLib

2.1.6

### Fabric API

0.76.0

### Fabric Loader

0.14.17

### Minecraft

1.19.2

### Relevant log output

```shell
Description: Exception generating new chunk

java.lang.IllegalStateException: Feature order cycle found, involved sources: [Reference{ResourceKey[minecraft:worldgen/biome / betterend:chorus_forest]=net.minecraft.class_1959@1d7e613}, Reference{ResourceKey[minecraft:worldgen/biome / betterend:old_bulbis_gardens]=net.minecraft.class_1959@40b0df0f}]
	at net.minecraft.class_7510.method_44210(class_7510.java:100)
	at org.betterx.bclib.api.v2.generator.BCLChunkGenerator.lambda$rebuildFeaturesPerStep$5(BCLChunkGenerator.java:103)
	at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:183)
	at net.minecraft.class_2794.method_12102(class_2794.java:397)
	at net.minecraft.class_2806.method_20613(class_2806.java:145)
	at net.minecraft.class_2806.method_12154(class_2806.java:292)
	at net.minecraft.class_3898.method_17225(class_3898.java:773)
	at com.mojang.datafixers.util.Either$Left.map(Either.java:38)
	at net.minecraft.class_3898.method_17224(class_3898.java:771)
	at java.base/java.util.concurrent.CompletableFuture$UniCompose.tryFire(CompletableFuture.java:1150)
	at java.base/java.util.concurrent.CompletableFuture$Completion.run(CompletableFuture.java:482)
	at net.minecraft.class_3900.method_17634(class_3900.java:62)
	at net.minecraft.class_3846.method_16907(class_3846.java:91)
	at net.minecraft.class_3846.method_16900(class_3846.java:146)
	at net.minecraft.class_3846.run(class_3846.java:102)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

### Other Mods

```shell
Mod:
Oh the Biomes Youll Go 2.0.1.1
Better Nether 7.1.3
Optifine 1.19.2_HD_U_I1 (Tested without this and the game still crashed but without giving me a detailed reason)
Optifabric 1.13.16 (This was also removed when I removed Optifine to test)

Libraries:
Corgilib 1.0.0.32
Geckolib 3.1.40
Terra Blender 2.0.1.136
```

**Comment by begriefed** [comment]
Current "fix" is to set

config > betterend > biomes.json

"chorus_forest": {
      "enabled [default: true]": false

I'm not sure what features this will break, but at least the end island will generate for a dragon fight until an update can fix the issue of https://misode.github.io/guides/feature-order-cycle/

**Comment by quiqueck** [comment]
Fixed in 2.1.5
